### PR TITLE
Commiting the select parent , add leads workflow

### PR DESCRIPTION
**Pull request by ishrartx**

### CODE REVIEW

File: src/test/java/step_definitions/SalesForceMarketing.java [hunk 1]
```diff
@@ -34,75 +35,105 @@ public void readTheTestdataFromExcelFile(String arg1, String arg2) {
...
+        CampaignUtil.clickCampaignTab("click on the compaign tab");
     }
 
     @Then("user create new Campaign")
-    public void create_New_Compaign(){
-        try{
-            CampaignUtil.createCampaign(dataMap.get("ComapaignName"));
+    public void create_New_Compaign() {
+        try {
+            CampaignUtil.createCampaign(dataMap.get("CamapaignName"));
```
Comment (snaphray) [inline src/test/java/step_definitions/SalesForceMarketing.java]: Please check the spelling of the column Name 'CamapaignName'

Comment (ishrartx) [reply src/test/java/step_definitions/SalesForceMarketing.java:66]: I have checked it

Comment (ishrartx) [reply src/test/java/step_definitions/SalesForceMarketing.java]: Done

**Review by snaphray — APPROVED** [review]
Code looks fine

**Pull request merged**